TaylorFramework/Modules/caprices/ProccessingMessage/Options/ExcludesFileOption.swift

```diff
@@ -35,7 +36,7 @@ struct ExcludesFileOption: ExecutableOption {
         var excludePaths = [String]()
         do {
             excludePaths = try ExcludesFileReader().absolutePathsFromExcludesFile(path, forAnalyzePath: analyzePath)
-        } catch CommandLineError.ExcludesFileError(let errorMsg) {
+        } catch FileReaderError.ExcludesFile(let errorMsg) {
             errorPrinter.printError(errorMsg)
             return nil
         } catch { return nil }
```

Comment:
if you move the return of `excludePaths` to `do` block, you can move `return nil` out of both `catch` blocks.